lib/adapter/external/manga_kakalot.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Ewbscp.Adapter.External.MangaKakalot do
+  use Tesla
+
+  plug(Tesla.Middleware.BaseUrl, "https://ww7.mangakakalot.tv")
+
+  # returns the page with the search results (this would be a list of links to be presented to the user)
```

Comment:
Let's use [@doc](https://hexdocs.pm/elixir/1.12/writing-documentation.html) on all the external adapters? WDYT?

Reply:
This comment probably is just temporary, but I like the idea of having a @doc saying like:
```
Fetches a manga page based on its title from "https://ww7.mangakakalot.tv".

Returns `{:ok, body}`
```